radixdlt/src/main/java/com/radixdlt/network/consensus/SimpleEventCoordinatorNetwork.java

```diff
@@ -15,29 +15,31 @@
  *  language governing permissions and limitations under the License.
  */
 
-package com.radixdlt.network;
+package com.radixdlt.network.consensus;
```

Comment:
Seems like this should actually be a part of `middleware` module because it depends on and glues things between `com.radixdlt.network` and `com.radixdlt.consensus`.

Reply:
OK, going to move to com.radixdlt.middleware2.network for now then.